build-system/pr-check/module-tests.js

```diff
@@ -27,13 +27,13 @@ const {
   startTimer,
   stopTimer,
   timedExecOrDie: timedExecOrDieBase,
-  downloadEsmDistOutput,
-  downloadDistOutput,
+  downloadModuleOutput,
+  downloadNomoduleOutput,
```

Comment:
Why are we downloading nomodule files in `module-tests.js`? 🤔

Reply:
Good question. Module tests need to make sure that when both nomodule and module binaries are capable of being served, it picks and uses only the module output 😃